src/core/or/circuitpadding.c

```diff
@@ -1104,6 +1104,24 @@ circpad_new_consensus_params(const networkstatus_t *ns)
          CIRCWINDOW_START_MAX, 0, 50*CIRCWINDOW_START_MAX);
 }
 
+/**
+ * Return true if padding is allowed by torrc and consensus.
+ */
+STATIC bool
+circpad_is_padding_allowed(void)
```

Comment:
No need to have `circpad` as a function name prefix since it's a private function to this file.

Reply:
a151032662efdb2421187d83dade96f5a1634499 does this (and fixes the CI error)